lib/notifier.js

```diff
@@ -73,60 +116,29 @@ function buildBaseData(extra) {
   });
 
   data.server = {
-    host: SETTINGS.host,
+    host: this.settings.host,
     argv: process.argv.concat(),
     pid: process.pid
   }
 
-  data.server.host = SETTINGS.host;
+  data.server.host = this.settings.host;
 
-  if (SETTINGS.branch) {
-    data.server.branch = SETTINGS.branch;
+  if (this.settings.branch) {
+    data.server.branch = this.settings.branch;
   }
-  if (SETTINGS.root) {
-    data.server.root = SETTINGS.root;
+  if (this.settings.root) {
+    data.server.root = this.settings.root;
   }
 
   return data;
-}
-
-
-function buildErrorData(baseData, err, callback) {
-  parser.parseException(err, function (e, errData) {
-    if (e) {
-      return callback(e);
-    }
-    baseData.body.trace = {
-      frames: errData.frames,
-      exception: {
-        class: errData['class'],
-        message: errData.message
-      }
-    };
-
-    callback(null);
-  });
-}
-
-
-function charFill(char, num) {
-  var a, x;
-
-  a = [];
-  x = num;
-  while (x > 0) {
-    a[x] = '';
-    x -= 1;
-  }
-  return a.join(char);
-}
+};
 
 
-function scrubRequestHeaders(headers, settings) {
+Notifier.prototype._scrubRequestHeaders = function(headers, settings) {
```

Comment:
I would remove the second param here since we will always want to use the settings from the Notifier instance.